README.md

```diff
@@ -145,7 +144,41 @@ Git Operator pushes SDP entities on Kubernetes to a Git repository and create pu
          vendor: gitlab
          mainBranch: main
          baseBranch: main
-         repository: appgate/gitlab-example
+         repository: <GITLAB_REPOSITORY>
+   ```
+   ```shell
+   helm install gitlab-operator appgate/sdp-operator --values gitlab-operator.yaml --namespace sdp-system
+   ```
+
+#### GitLab (Self-Hosted)
+1. Generate a file containing the host key fingerprint of the self-hosted GitLab
+   ```shell
+   ssh-keyscan <GITLAB_HOSTNAME> > <SSH_HOST_KEY_FINGERPRINT_PATH>
+   ```
+
+2. Create a secret containing SSH key and GitLab token
+   ```shell
+   kubectl create secret generic gitlab-operator-secret \
+       --from-literal=gitlab-token="<GITLAB_TOKEN>" \
+       --from-file=git-ssh-key="<SSH_KEY_PATH>" \
+       --from-file=git-ssh-host-key-fingerprint="<SSH_HOST_KEY_FINGERPRINT_PATH>"
+   ```
+
+3. Install the Git Operator for self-hosted GitLab
+   ```yaml
+   sdp:
+     operators:
+       - git-operator
+     gitOperator:
+       version: v18
+       secret: gitlab-operator-secret
+       git:
+         vendor: gitlab
+         mainBranch: main
+         baseBranch: main
+         repository: <GITLAB_REPOSITORY>
+         hostname: <GITLAB_HOSTNAME>
+         strictHostKeyChecking: true
```

Comment:
Is that the default as well?

Reply:
Yes. Any values that is hardcoded in values.yaml is the default. Maybe I'll remove it from this example to avoid confusion.